lib/galaxy/datatypes/interval.py

```diff
@@ -386,14 +386,18 @@ class Bed(Interval):
     MetadataElement(name="chromCol", default=1, desc="Chrom column", param=metadata.ColumnParameter)
     MetadataElement(name="startCol", default=2, desc="Start column", param=metadata.ColumnParameter)
     MetadataElement(name="endCol", default=3, desc="End column", param=metadata.ColumnParameter)
-    MetadataElement(name="strandCol", desc="Strand column (click box & select)", param=metadata.ColumnParameter, optional=True, no_value=0)
+    MetadataElement(name="strandCol", default=0, desc="???Strand column (click box & select)", param=metadata.ColumnParameter, optional=True, no_value=0)
+    MetadataElement(name="strandColTest", default=0, desc="Strand column (click box & select)", param=metadata.ColumnParameter, optional=True, no_value=0)
     MetadataElement(name="columns", default=3, desc="Number of columns", readonly=True, visible=False)
     MetadataElement(name="viz_filter_cols", desc="Score column for visualization", default=[4], param=metadata.ColumnParameter, optional=True, multiple=True)
     # do we need to repeat these? they are the same as should be inherited from interval type
 
     def set_meta(self, dataset, overwrite=True, **kwd):
         """Sets the metadata information for datasets previously determined to be in bed format."""
         i = 0
+        log.error(f"{type(dataset.metadata.strandCol)}")
+        log.error(f"dataset.metadata.strandCol={dataset.metadata.strandCol}")
```

Comment:
@mvdbeek do you have an idea why `strandCol` is here `None` but `strandColTest` is `0` despite that both have `default=0`?

Reply:
No idea, I don't really understand all that weird stuff going on in MetadataElement

Reply:
Ahh. I was testing `python "metadata/set.py"` interactively for an upload job.. seems that default metadata values are stored on disk (no idea how they get there). So changing the default in the code had no effect in my manual test.